Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -2528,6 +2545,10 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
             cell = labelAndSwitchCell;
         }
+        else if (row == LABS_ENABLE_LIVE_LOCATION_SHARING && BuildSettings.liveLocationSharingEnabled) // Hide live location lab setting until it's ready to be release
```

Comment:
`BuildSettings.liveLocationSharingEnabled` check is not needed here. See comment above.